api_bonus.yaml

```diff
@@ -0,0 +1,144 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Bonus API
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  "/bonus/vacanze/request":
+    post:
+      operationId: startBonusEligibilityCheck
+      summary: Start bonus eligibility check (ISEE)
+      responses:
+        '202':
+          description: Request accepted.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          headers:
+            Location:
+              type: string
+              description: |-
+                Location (URL) of created request resource.
+                A GET request to this URL returns the request status and details.
+          examples:
+            application/json:
+              id: 01BX9NSMKVXXS5PSP2FATZMYYY
+
+    get:
+      operationId: getBonusEligibilityCheck
+      summary: Get eligibility (ISEE) check information for user's bonus
+      responses:
+        '202':
+          description: Processing request.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          examples:
+            application/json:
+              id: 01BX9NSMKVXXS5PSP2FATZMYYY
+        '200':
+          description: Request processed.
+          schema:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The identifier of the created request.
+          examples:
+            application/json:
+              # REQUEST (isee): NO_DATA, ELIGIBLE, INELIGIBLE, NOT_FOUND
```

Comment:
perhaps a short explanation of the different states would be useful, i.e. what's the difference between `NO_DATA` and `NOT_FOUND` ?

perhaps we should use an enum here?

Reply:
yes of course, this is a rough draft :)